pages/learn/manage/organizations.md

```diff
@@ -0,0 +1,104 @@
+
+* **Administrator**: Full access to the organization, its applications, and all devices inside them. Administrators can set roles for incoming members, group them into teams, and provide application access to those teams with granular control over which roles are being assigned at the application level.
+
+* **Member**: Members are provided with read-only access to an organization with explicit access to its applications and teams. Administrators assign roles on the application level. Members have no visibility over other applications and teams that they haven't been assigned access to.
+
+### Inviting members to an organization
+
+With your organization created, let's invite some members. Only the administrator can invite users to an organization. To invite, click the `Manage members` button on the organization summary dashboard. On the page that opens, click the `Invite member` button to open the `Invite organization member` modal. Here, you can either add users with their email address or with their balena username after assigning them roles in your organization. Click `Add member` to invite the user to your organization.
+
+<img alt="Invite Organization Member Modal" src="/img/common/app/invite_org_member.png">
+
+Depending on the role assigned, they will be able to access, see, and manage both apps and other members inside the organization. To change a member's role, navigate to the `Organization Members` tab in the sidebar. Next, change the `Organization role` of the desired user using the dropdown list against their username.
+
+<img alt="Change Organization Member Roles" src="/img/common/app/modify_org_roles.png" width="100%">
+
+
+## Creating teams in your organization
+
+With organizations, you can group your existing members to form teams as well. Teams are a robust way to manage members by their function, and bulk provides access to applications as needed. Only administrators are allowed to create a team. To form a team, navigate to the `Organization Teams` tab in the sidebar. On the page that opens, click the `Create Team` button and enter the team's name that is being created.
+
+<img alt="Create Team in Organization" src="/img/common/app/create_team.png" width="100%">
+
+When the team is created, you can start adding members and applications to that specific team through the `Manage members` and `Manage applications` buttons, respectively.
+
+<img alt="Organization Team Summary" src="/img/common/app/org_team_summary.png" width="100%">
+
+
+## Creating new applications under your organization
+
+With members invited to the organization, let's get started on creating new applications under an organization. Navigate to the `Organization Applications` tab in the sidebar to find the `Create application` button for creating a new application under your organization. Learn more about creating an application from the [getting started](getting-started) guide.
```

Comment:
I may be misreading the document, but it feels like it can't decide whether its function is to provide reference-level documentation or be a kind of "getting started with orgs" guide. It's perhaps okay as a start, but I personally think we should revisit soon after release.

Reply:
@dfunckt I started out with a purely referential piece of documentation complementing the blog we are writing but I agree in the end it seems to have a more a leaner, getting started guide tone.